src/components/safe-apps/SafeAppIcon/index.tsx

```diff
@@ -1,17 +1,19 @@
 import { type ReactElement, memo } from 'react'
 
-const getIframeContent = (src: string): string => {
-  return `<style>
-    html, body {
-      padding: 0;
-      margin: 0;
-      background-color: transparent;
-      height: 100%;
-    }
-    body {
-      background: url('${src}') center center/contain no-repeat;
-    }
-  </style>`
+const APP_LOGO_FALLBACK_IMAGE = `/images/apps/app-placeholder.svg`
+
+const getIframeContent = (url: string, alt: string, width: number, height: number): string => {
+  return `
+     <body style="margin: 0;">
+       <img src=${url} alt=${alt} width="${width}" height="${height}" />
```

Comment:
I would add `style="display: block"` to avoid a line underneath.

Reply:
What line? I didn't see anything unwanted in all browsers I tested (chrome, safari desktop, firefox)

Reply:
I remember you mentioned it in the slack convo but when working on it I didn't see a need for it

Reply:
This line:
<img width="657" alt="Screenshot 2023-01-11 at 08 46 35" src="https://user-images.githubusercontent.com/381895/211747558-3442cb38-63d8-45ce-979c-5f03fd7eb431.png">
Inline elements such as img have a line-height that are bigger than the height of the image itself.

Reply:
Doesn't really matter in this case tho, because the iframe itself is 40x40.